WRN-1663: Add wheelEventInterval in Slider to prevent firing wheel event too frequently (#993)
enactjs/sandstone: CHANGELOG.md

### Added

- `sandstone/ImageItem` public classname `imageIcon`

> - `sandstone/Slider` prop `wheelEventInterval` to control speed of the wheel on the slider

Reviewer: duplicated component name "slider"

```suggestion
- `sandstone/Slider` prop `wheelInterval` to throttle the wheel input
```

Author: It's fixed.
